src/main/java/minicraft/core/io/Localization.java

```diff
@@ -1,28 +1,43 @@
 package minicraft.core.io;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Locale;
-
-import org.jetbrains.annotations.NotNull;
-
 import minicraft.core.Game;
+import minicraft.saveload.Load;
 import minicraft.util.Logging;
-
+import org.jetbrains.annotations.NotNull;
 import org.json.JSONObject;
 import org.tinylog.Logger;
 
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Locale;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
 public class Localization {
 
 	public static final Locale DEFAULT_LOCALE = Locale.US;
+	public static final Locale DEBUG_LOCALE = Locale.ROOT; // This locale is used for debugging;
 
 	private static final HashMap<Locale, HashSet<String>> knownUnlocalizedStrings = new HashMap<>();
 	private static final HashMap<String, String> localization = new HashMap<>();
+	private static final HashMap<String, String> fallbackLocalization = new HashMap<>();
 
 	private static Locale selectedLocale = DEFAULT_LOCALE;
 	private static final HashMap<Locale, ArrayList<String>> unloadedLocalization = new HashMap<>();
 	private static final HashMap<Locale, LocaleInformation> localeInfo = new HashMap<>();
+	private static final Set<String> DEFAULT_LOCALIZATION_OF_DEFAULT_LOCALE;
+
+	static {
+		try {
+			DEFAULT_LOCALIZATION_OF_DEFAULT_LOCALE =
+				new JSONObject(String.join("", Load.loadFile("/assets/localization/en-us.json"))).keySet();
+		} catch (IOException e) {
+			throw new RuntimeException(e); // CRITICAL ERROR (GAME ASSETS)
+		}
+	}
```

Comment:
It's probably better to put this in a constructor. I think we are going to need to move away from static variables to make the code easier to understand.

Reply:
That means we would need to move static fields into object fields for all the classes.

Reply:
Yeah, but constants generally don't need to be changed. So only the static fields.

Reply:
Also noticed that the name of this field is wrong. It it should be camelCase.

Reply:
Public static fields are all constants, so they should be UPPER_SNAKE_CASE.

Reply:
No, that set is not a constant.

Reply:
Right, and that part you commented is removed now.